deploy/008-deploy-vaults.ts

```diff
@@ -12,20 +12,26 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     args: [],
     log: true,
     autoMine: true,
+    skipIfAlreadyDeployed: true,
+    maxFeePerGas: "200000000",
```

Comment:
Should this value depends on the network, as we made in other places?

Reply:
[51c0702](https://github.com/VenusProtocol/venus-protocol/pull/506/commits/51c070278168a0a65789c3b9e6320a1f57d5876c)